docs/faq/README.md

```diff
@@ -52,14 +52,19 @@ System Cron:
 * Check if the **oc_jobs** table has a **reserved_at** entry with a value other than 0. If it does for whatever reason, set it to 0. You can check this by executing:
 
   ```sql
-  SELECT reserved_at FROM oc_jobs WHERE (argument = '["OCA\\News\\Cron\\Updater","run"]' OR class = 'OCA\\News\\Cron\\Updater');
+  SELECT reserved_at FROM oc_jobs WHERE (argument = '["OCA\News\Cron\Updater","run"]' OR class = 'OCA\News\Cron\Updater');
   ```
+_For newer News versions please also use_
+SELECT reserved_at FROM oc_jobs WHERE (argument = '["OCA\News\Cron\UpdaterJob","run"]' OR class = 'OCA\News\Cron\UpdaterJob');
 
  and reset it by executing
 
   ```sql
-  UPDATE oc_jobs SET reserved_at = 0 WHERE (argument = '["OCA\\News\\Cron\\Updater","run"]' OR class = 'OCA\\News\\Cron\\Updater');
+  UPDATE oc_jobs SET reserved_at = 0 WHERE (argument = '["OCA\News\Cron\Updater","run"]' OR class = 'OCA\News\Cron\Updater');
   ```
+  
+  _For newer News versions please also use_
+  UPDATE oc_jobs SET reserved_at = 0 WHERE (argument = '["OCA\News\Cron\UpdaterJob","run"]' OR class = 'OCA\News\Cron\UpdaterJob');
```

Comment:
Same as above.
I would suggest to just append `OR class = 'OCA\News\Cron\UpdaterJob'` to the above query since it will also work on older NC or News versions.

Reply:
If I past the original command to my SQL workplace (Eclipse + DBeaver) it won't work.

Reply:
Same here with `UPDATE oc_jobs SET reserved_at = 0 WHERE (argument = '["OCA\\News\\Cron\\Updater","run"]' OR class = 'OCA\\News\\Cron\\Updater' OR class = 'OCA\\News\\Cron\\UpdaterJob');` .. for you with PgSQL probably without the `\\`